README.md

```diff
@@ -104,48 +104,71 @@ julia> setindex(a, 1.337, 2)
 
 ## Operations
 
-The symbol `*` is overloaded for double contractions between fourth and second order tensors, and single contractions between two second order tensors or between a second and first order tensor.
-
-### Double contractions
-
-Double contractions contracts the two most inner "legs" of the tensors. The result of a double contraction between a tensor of order `n` and a tensor with order `m` gives a tensor with order `m + n - 4`. The symbol `⊡`, written `\boxdot` is overloaded for double contraction. The reason `:` is not used is because it does not have the same precedence as multiplication.
+### Scaling
+The symbol `*` can be used for multiplication between a number and a tensor, i.e. scaling of the tensor `A` with scalar `a`.
```

Comment:
I don't think this need to be documented. You can also divide with a scalar and add a constant etc. Jobbigt to document and pretty obvious.